src/metabase/pulse/render/common.clj

```diff
@@ -94,13 +94,28 @@
    (or (ui-logic/y-axis-rowfn card data)
        second)])
 
-(defn non-nil-rows
+(defn coerce-bignum-to-int
+  "Graal polyglot system (not the JS machine itself, the polyglot system)
+  is not happy with BigInts or BigDecimals.
+  For more information, this is the GraalVM issue, open a while
+  https://github.com/oracle/graal/issues/2737
+  Because of this unfortunately they all have to get smushed into normal ints and decimals in JS land."
+  [row]
+  (for [member row]
+    (cond
+      ;; this returns true for bigint only, not normal int or long
+      (and (not (int? member)) (integer? member))
```

Comment:
Would it be more straightforward to do something like `(instance? clojure.lang.BigInt member)`?